spec/models/carrier_spec.rb

```diff
@@ -14,7 +14,8 @@
       color: 'blue',
       home_location_id: washington.id,
       current_location: lancaster,
-      category_id: category.id
+      category_id: category.id,
```

Comment:
Atm we only have 1 carrier fixture, we may wanna add few more base on the statuses and use them in the spec instead of reaching out to `ActiveRecord#new`

Reply:
Ah, fair enough. I'll add those.

Reply:
I added the additional fixtures. However, it looks like we're newing things up here because we're checking validation which required a new model to check properly.

Reply:
We can use fixture for that something like this:
```ruby
let(:carrier) { carriers(:carrier) }

it "should have valid fixture" do
  expect(carrier).to be_valid
end
```

Then we can also remove specs on the line 57 and 61. Every fixture we have should be wit a valid data, so there is no point on testing basic rails validations such as "presence, length, validations", all of those are tested in the Rails itself 😃 , only if we have some more complex/custom validations